packages/lab/src/Canvas/stories/Canvas.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from "react";
+import { css } from "@emotion/css";
+import { Meta, StoryObj } from "@storybook/react";
+import { BackgroundVariant } from "reactflow";
+import {
+  HvButton,
+  HvDropDownMenu,
+  HvIconButton,
+  HvInlineEditor,
+  HvLoading,
+  theme,
+} from "@hitachivantara/uikit-react-core";
+import {
+  Backwards,
+  Calendar,
+  DataSource,
+  Plane,
+  Pod,
+  Redo,
+  Schema,
+  Undo,
+  User,
+} from "@hitachivantara/uikit-react-icons";
+import {
+  HvCanvasPanel,
+  HvCanvasTab,
+  HvCanvasToolbar,
+  HvFlow,
+  HvFlowBackground,
+  HvFlowMinimap,
+} from "@hitachivantara/uikit-react-lab";
+
+import { ListView } from "./ListView/ListView";
+import { eds, nds, nodeTypes } from "./nodes";
+import { TreeView } from "./TreeView";
+
+const meta: Meta = {
+  title: "Lab/Canvas",
+};
+export default meta;
+
+export const Main: StoryObj = {
+  args: {
+    title: "Semantic Model Name",
+  },
+  argTypes: {
+    classes: { control: { disable: true } },
+  },
+  render: () => {
+    const [tabs, setTabs] = useState<HvCanvasTab[]>([]);
+    const [selectedTab, setSelectedTab] = useState<string>("1");
+
+    const classes = {
+      separator: css({
+        height: 30,
+        width: 1,
+        backgroundColor: theme.colors.atmo4,
+        margin: `0 ${theme.space.xs}`,
+      }),
+    };
+    const backButton = (
+      <HvButton aria-label="Back" icon>
+        <Backwards />
+      </HvButton>
+    );
+
+    const title = (
+      <HvInlineEditor value="Semantic Model Name" variant="title4" />
+    );
+
+    const Separator = () => <div className={classes.separator} />;
+
+    const renderIcon = (Icon: React.ElementType) => () => <Icon />;
+
+    useEffect(() => {
```

Comment:
Can't `tabs` be static instead of setting it in a `useEffect`? I thought the bug we had where we needed to do that was solved in recent versions o Storybook. This way we don't need the `HvLoading` too.